packages/repository/src/legacy-juggler-bridge.ts

```diff
@@ -195,6 +196,15 @@ export class DefaultCrudRepository<T extends Entity, ID>
     return ensurePromise(this.modelClass.exists(id, options));
   }
 
+  async execute(
+    query: string | AnyObject,
+    // tslint:disable:no-any
+    parameters: AnyObject | any[],
+    options?: Options,
+  ): Promise<AnyObject> {
+    throw new Error('Not implemented');
+  }
```

Comment:
The addition of `execute` method makes a lot of sense to me, I think this should be added to loopback-next in a standalone PR, including tests etc.